app/assets/stylesheets/stream.scss

```diff
@@ -1,6 +1,25 @@
 .stream_container {
-  #publisher {
-    margin-bottom: 15px;
+  .stream-title {
+    margin: 12px 0;
+  }
+}
+
+.main-stream-publisher {
+  padding: 0;
+
+  .avatar {
+    height: 50px;
+    width: 50px;
+  }
+
+  .publisher {
+    margin-left: 65px;
+  }
+}
+
+@media(max-width: $screen-xs) {
```

Comment:
It works :) Waiting for the build to complete then I'll commit and you'll be able to do the final review.